src/dom/get-visible-y-from-root.ts

```diff
@@ -0,0 +1,47 @@
+import { dom } from '../';
+import { domVectorf } from './dom-vectorf';
+
+const ignoredPositions = new Set(['fixed', 'absolute']);
```

Comment:
Where is this being used?

Reply:
It isn't anymore, sorry for missing this.

There was code for getting what the position of a sticky element would be if it wasn't stickied. This wasn't an immediate need so I removed it from this PR but this variable squeaked through unnoticed.

Removed.